web/pdf_viewer.js

```diff
@@ -2164,29 +2199,31 @@ class PDFViewer {
   /**
    * Increase the current zoom level one, or more, times.
    * @param {number} [steps] - Defaults to zooming once.
+   * @param {Object|null} [options]
    */
-  increaseScale(steps = 1) {
+  increaseScale(steps, options = null) {
```

Comment:
Why are you removing the `scale = 1` default, since that leads to a number of otherwise unnecessary changes throughout the code-base?